### PR TITLE
Change IR from light to switch docs

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Change IR from light to switch, see PR: https://github.com/home-assistant/core/pull/105916

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/105916
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -173,6 +174,8 @@ Depending on the supported features of the camera, switch entities are added for
 - FTP upload
 - HDR*
 
+When **Infra red lights in night mode** entity is OFF always OFF, when ON IR LEDs will be on when the camera is in night vision mode, see the **Day night mode** select entity.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
When the **Infrared lights in night mode** entity is set to OFF, the infrared LEDs are always OFF. When the **Infrared lights in night mode** entity is set to ON, the infrared LEDs will be on when the camera is in night vision mode. For more information, see the **Day night mode** select entity.
```
hm, not sure I understand this. Please correct me if I got this wrong.

File: source/_integrations/reolink.markdown [hunk 2]
```diff
@@ -160,6 +160,7 @@ In some camera models, there is a delay of up to 5 seconds between the turn-off
 
 Depending on the supported features of the camera, switch entities are added for:
 
+- Infra red lights in night mode
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
- Infrared lights in night mode
```

**Review by c0ffeeca7 — COMMENTED** [review]
Tweaks

File: source/_integrations/reolink.markdown (hunk 1 above)
Comment (starkillerOG) [reply source/_integrations/reolink.markdown]: No you got it.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @starkillerOG :+1:

**Pull request merged**